src/tests/ftest/launch.py

```diff
@@ -1161,7 +1168,16 @@ def rename_logs(avocado_logs_dir, test_file):
     test_name = get_test_category(test_file)
     test_logs_lnk = os.path.join(avocado_logs_dir, "latest")
     test_logs_dir = os.path.realpath(test_logs_lnk)
-    new_test_logs_dir = "{}-{}".format(test_logs_dir, test_name)
+
+    if args.jenkinslog:
+        new_test_logs_dir = os.path.join(avocado_logs_dir, test_file)
+        try:
+            os.makedirs(new_test_logs_dir)
+        except OSError as error:
+            print("Error mkdir {}: {}".format(new_test_logs_dir, error))
```

Comment:
I think it might be better to do this:
            print("Error mkdir {}: {}".format(new_test_logs_dir, error))
            new_test_logs_dir = "{}-{}".format(test_logs_dir, test_name)

So in this case, if it failed to makedirs, it will not fail out but attempt to run with the -r flow, and hopefully we can get still test results/logs, if issue is isolated to makedirs.